### PR TITLE
Move Estimate filter into Filters tabs popup

### LINKED ISSUE #1696: Move Estimate filter into Filters tabs popup
After #1235

### LINKED ISSUE #1732: Fix Estimate Picker sizes
Current text sizes are too small 🔎
Screenshot 2023-09-28 at 14 31 01

### PR DESCRIPTION
**Pull request by asabotovich**
## PR includes

- [x] Feature

## Related issues

Resolve #1696 
Resolve #1732 

## QA Instructions, Screenshots, Recordings

https://github.com/taskany-inc/issues/assets/9020793/3359b7ed-af61-460a-881a-3af7fdd5c059

### CODE REVIEW

File: src/components/Estimate/Estimate.tsx [hunk 1]
```diff
@@ -1,63 +1,88 @@
...
+`;
+
+const StyledWarningWrapper = styled.div`
+    display: flex;
+    align-items: center;
+    gap: ${gapXs};
+`;
+
+const StyledWarningCircle = styled.div`
+    width: 12px;
```
Comment (awinogradov) [inline src/components/Estimate/Estimate.tsx:28]: Sorry, but it's impossible to use custom sizes for icons... U must use @taskany/icons instead. Thanks for understanding.

**Review by awinogradov — CHANGES_REQUESTED** [review]

File: src/components/Estimate/Estimate.tsx [hunk 2]
```diff
@@ -1,63 +1,88 @@
...
-    placeholder,
-    value,
-    placement,
-    error,
-    renderTrigger,
-    onChange,
-    onClose,
-}) => {
-    const [readOnly, setReadOnly] = useState({ year: true, quarter: true, date: true });
+export const getReadOnlyFileds = (type: EstimateValue['type']) => {
```
Comment (LamaEats) [inline src/components/Estimate/Estimate.tsx:46]: Fileds -> Fields

File: src/components/Estimate/Estimate.tsx [hunk 3]
```diff
@@ -84,73 +108,46 @@ export const Estimate: React.FC<EstimateProps> = ({
...
+                title={tr('Year title')}
+                clue={tr('Year clue')}
+                value={year}
+                readOnly={readOnly.year}
+                onChange={setYear}
+                setReadOnly={setReadOnly}
+            />
+            <EstimateQuarter
+                title={tr('Quarter title')}
+                clue={`${tr('Quarter clue')} ${getQuarterFromDate(new Date())}.`}
```
Comment (LamaEats) [inline src/components/Estimate/Estimate.tsx:138]: why is not translated template string formatted through `tr.raw` function?

**Review by LamaEats — APPROVED** [review]

**Comment by asabotovich** [comment]
Fix text size and humanize translations
Screenshot 2023-09-28 at 16 03 12

**Comment by asabotovich** [comment]
Design fixes

Screenshot 2023-09-28 at 17 22 59

**Review by awinogradov — APPROVED** [review]

**Comment by asabotovich** [comment]
Resovle: #1734

**Pull request merged**